<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeWindow</class>
 <widget class="QDialog" name="EmployeeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>151</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EmployeeWindow — The Dollar Tracker</string>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <item>
    <widget class="QLabel" name="lbl">
     <property name="text">
      <string>Welcome</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnViewShifts">
     <property name="text">
      <string>View My Shifts</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnIn">
     <property name="text">
      <string>Clock In</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnOut">
     <property name="text">
      <string>Clock Out</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblStatus">
     <property name="text">
      <string>Status: …</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSignOut">
     <property name="text">
      <string>Sign Out</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
